src/main/java/com/neonlab/product/service/SuggestionService.java

```diff
@@ -29,6 +32,37 @@ public ApiOutput<SuggestionDto> createSuggestion(SuggestionDto suggestionDto) th
         return new ApiOutput<>(HttpStatus.OK.value(), "Thank You for Giving me Suggestion",suggestionDtos);
     }
 
+    public ApiOutput<List<SuggestionDto>> fetchSuggestion() throws InvalidInputException, ServerException {
```

Comment:
*Naming Convention*

- The method `fetchSuggention` is in *SuggetionService*.  
- Motive of giving the name is to communicate the operation it's doing. 
- Here, `fetchSuggestion` tells to get the list of suggestions. 
- The same thing can all be communicated by `suggestionService.fetch`

In an Office, a person's work is only to maintain the in and outs of employees in a record. 
If you want him to give the record, you will say `person.record` and not `person.inAndOutRecord`. But if you are asking him some other record that is out of his `concern` then you need to explicitly define which like `person.someOtherRecord`.

cc- @sjha24

Reply:
Service layer should not give ApiOut put . This is the concern of the Api layer and not service layer.

Give normal output. Attach that to the ApiOutput in the API layer.